src/components/Menu/EmojiPickerAction.vue

```diff
@@ -0,0 +1,58 @@
+<!--
+  - @copyright Copyright (c) 2022 Vinicius Reis <vinicius@nextcloud.com>
+  -
+  - @author Vinicius Reis <vinicius@nextcloud.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+<template>
+	<EmojiPicker class="entry-action entry-action__emoji"
+		:data-text-action-entry="actionEntry.key"
+		@selectData="addEmoji">
+		<button v-tooltip="actionEntry.label"
```

Comment:
We could switch to the `@nextcloud/vue` button here in order to avoid custom styling from https://github.com/nextcloud/text/pull/2439/files#diff-ad1f213fdd71dfbb7fd8df37ade6a17b7c16e70098218a4b76cce502ee857eabR18